scripts/test-template.sh

```diff
@@ -25,7 +25,13 @@ echo '--- pnpm build'
 pnpm build
 
 echo '--- pnpm pack'
+# I'm sure there's a better way to do this
+eslint_config_skuba_tar="$(pwd)/packages/eslint-config-skuba/$(cd packages/eslint-config-skuba && pnpm pack | grep -o 'eslint-config-skuba-.*\.tgz')"
```

Comment:
Hmmmm why is this needed again?

Reply:
Integration tests were annoyingly running against the published version of eslint-config-seek, which horribly broke everything. Idk how to get it to do sane things